### PR TITLE
Allows ChemMasters to make multiple bottles at once

### PR DESCRIPTION
**Pull request by swindly**
:cl: Swindly
add: Added a button to the ChemMaster that dispenses the entire buffer to bottles.
/:cl:

The button now dispenses the entire buffer into bottles. Instead of dividing the buffer by a number set by the user like in patches and pills, the machine automatically uses the fewest number of bottles possible.

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 1]
```diff
@@ -262,21 +262,37 @@
...
-				P = new/obj/item/weapon/reagent_containers/glass/bottle(src.loc)
-
-			P.pixel_x = rand(-7, 7) //random position
-			P.pixel_y = rand(-7, 7)
-			P.name = trim("[name] bottle")
-			reagents.trans_to(P, P.volume)
+				var/amount = 1
+				var/vol_each = min(reagents.total_volume, 30)
+				if(text2num(many))
+					amount = min(max(round(input(usr, "Max 10. Buffer content will be split evenly.", "How many patches?", amount) as num|null), 0), 10)
```
Comment (optimumtact) [inline code/modules/reagents/chemistry/machinery/chem_master.dm]: use the clamp function, not a chained min and max

**Comment by Bawhoppen** [comment]
Chemmasters should require glass tbh

**Comment by chanoc1** [comment]
In repsponse to Bawhoppen, I agree, this is a nice QoL feature, but it needs something to stop shitters from creating a shitton of containers

**Comment by swindly** [comment]
The behavior has changed so that the maximum amount of bottles that can be made at once is 4, down from 10.

**Comment by optimumtact** [comment]
canUseTopic only checks if you're close if you specify the flag be_close=TRUE in the call iirc

**Pull request merged**